edk2toollib/uefi/edk2/parsers/buildreport_parser.py

```diff
@@ -140,7 +140,7 @@ def Parse(self):
                                 logging.debug("Parsing Mod: %s" % value)
                                 self.Name = value
                             elif(key == "module inf path"):
-                                self.InfPath = value
+                                self.InfPath = value.replace("\\", "/")
```

Comment:
@matthewfcarlson  this is the change needed for edk2 INF  path

Reply:
Looks good to me